[12-Sep-2022 00:16:26 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(37): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 00:16:45 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(14): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 00:17:19 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(14): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 00:17:25 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('bilanse')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:17:30 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('bilance')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:30:33 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:30:43 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:40:50 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:49:42 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:50:53 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:51:33 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('profile')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:53:10 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:53:44 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('show')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:53:49 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('profile')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 00:59:08 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:00:51 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:01:25 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:01:36 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('dadad')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:24 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('password/')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:24 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('password/css/cs...')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:27 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('password')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:29 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('password/')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:29 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('password/css/cs...')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:02:48 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:03:26 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('jjjjjh')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:10:58 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:11:03 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('profile')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:11:10 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:11:43 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('edit')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:11:48 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('show')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:11:52 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:12:07 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('icome')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:14:05 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:17:55 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:30:37 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:37:51 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:42:36 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:42:55 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:43:26 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:43:50 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('fsf')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:43:59 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('home')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:45:28 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('sdada')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:46:37 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('sdada')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:46:45 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:49:31 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:49:45 Europe/Warsaw] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('sdad')
#1 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php' on line 110
[12-Sep-2022 01:55:55 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:56:23 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/BalanceSheet.php(140): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Balance.php(31): App\Models\BalanceSheet::loadIncomeResults()
#3 [internal function]: App\Controllers\Balance->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('balance')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 01:56:37 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 02:01:42 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 02:01:58 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(22): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
[12-Sep-2022 02:03:59 Europe/Warsaw] Uncaught exception: 'ErrorException' with message 'Trying to get property 'id' of non-object'
Stack trace: #0 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php(44): Core\Error::errorHandler(8, 'Trying to get p...', '/home/klient.dh...', 44, Array)
#1 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Models/Expenses.php(24): App\Auth::getUserId()
#2 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Controllers/Expense.php(21): App\Models\Expenses::loadExpenseCategoriesData()
#3 [internal function]: App\Controllers\Expense->newAction()
#4 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Controller.php(51): call_user_func_array(Array, Array)
#5 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/Core/Router.php(101): Core\Controller->__call('new', Array)
#6 /home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/public_html/index.php(54): Core\Router->dispatch('expense')
#7 {main}
Thrown in '/home/klient.dhosting.pl/ppp/budget.arleta-madej.profesjonalnyprogramista.pl/App/Auth.php' on line 44
